MailSim/MailSim/ProvidersOM/MailStoreProviderOM.cs

```diff
@@ -189,9 +190,50 @@ public IAddressBook GetGlobalAddressList()
             if (_outlook != null && !_keepOutlookRunning)
             {
                 Console.WriteLine("Exiting Outlook");
- 
+
                 ((Outlook._Application)_outlook).Quit();
             }
         }
+
+
+        /// <summary>
+        /// Finds the Global Address List associated with the MailStore
+        /// </summary>
+        /// <param name="folderPath">parameter for FindFolder</param>
+        /// <returns>IMailFolder for the target of the event</returns>
+        /// In this case the Folder element is a path in Outlook. Each component of the path separated by '\'.
+        /// The first or default folder in the path, can be preceded by "\\" or nothing. If it's the only part of
+        /// the path, then it MUST be one of the default OL folders (see the schema for the EventMonitor operation). 
+        public IMailFolder FindFolder(string folderPath)
+        {
+            IMailFolder folder;
+            string backslash = @"\";
```

Comment:
I think it's actually:
            System.Char backslash = '\\';
but yeah. That should simplify.